WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1339,6 +1348,12 @@ extension EditableOrderViewModel {
             return Notice(title: Localization.errorMessageOrderCreation, feedbackType: .error)
         }
 
+        /// Returns an error notice when the given product ID cannot be found in local storage
+        ///
+        static func createOrderFromScannedProductErrorNotice(for ID: Int64) -> Notice {
```

Comment:
This is related to the comment above, why do we show an error if the product cannot be found in local storage? I don't think neither that's per se an error, nor we should communicate it to the user.

Reply:
It seems not needed indeed, specially since the latest refactor we expect to have a product or variation when this point is reached. The only error that could happen at this point is within the synchronizer, and we already have an implementation in place that will deal with this. I removed the method and the error case in the latests commits.

Reply:
Agree, removed on https://github.com/woocommerce/woocommerce-ios/pull/9768/commits/c38d3a4d644d3cb82da3c0f398b74a205cda1389